terraform/terraform.tfvars.example

```diff
@@ -1,6 +1,6 @@
-project = "clgcporg2-067"
+project = "clgcporg2-092"
```

Comment:
here is an example file so you must not change any "example" value to the real existing